axis_backend/apps/clients/models/tag.py

```
@@ -0,0 +1,79 @@
+        help_text="URL-friendly version of name"
+    )
+    color = models.CharField(
+        max_length=7,
+        default='#6B7280',
+        help_text="Hex color code for visual identification (e.g., '#10B981')"
+    )
+    description = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Purpose and usage guidance for this tag"
+    )
```

Ruff (0.14.6)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)